### DISCUSSION TITLE
help: Deep linking don't work on iOS

### DISCUSSION THREAD

**Discussion by JEricaM** [ionic-team/capacitor#4259] [Q&A]
### Capacitor Version
```
  @capacitor/ios 2.4.4
  @capacitor/cli 2.4.4
  @capacitor/core 2.4.4
  @capacitor/android 2.4.4
  @capacitor/electron not installed
```

### Platform(s)
iOS

### Current Behavior
I've implemented Deep linking for iOS using the [official Capacitor doc](https://capacitorjs.com/docs/guides/deep-links) (Android works good)
I've followed all the steps needed:

1. added "associated domains" capability
2. created apple-app-site-association file
3. uploaded the file on https
4. added on xcode the applinks:yourdomain.com that I need

I've also update my provisioning profile after I've added the new capability.

I've used [this tool](https://search.developer.apple.com/appsearch-validation-tool) as documentation says and I have one error:
"Error no apps with domain entitlements". What this error means? 

### Expected Behavior
Deep linking working well also on ios

### Code Reproduction
I don't know which code can be usefull for this problem.

### Other Technical Details

`npm --version` output: 6.9.0
 
`node --version` output: v10.16.0

`pod --version` output (iOS issues only): 1.10.0

Thank you

**Comment by jcesarmobile** [comment]
Moving to discussions as this is not a bug on Capacitor.

If the problem was that the app opens but doesn't trigger the App plugin events, then it could be a bug on Capacitor, but if the app is not opening, then it's some problem on the configuration of the deep links.
